src/main/java/com/dajudge/kindcontainer/K3sContainer.java

```diff
@@ -82,12 +93,34 @@ private String getDisabledComponentsCmdlineArg() {
     }
 
     @Override
-    public SELF withNodePortRange(int minPort, int maxPort) {
+    public SELF withNodePortRange(final int minPort, final int maxPort) {
         this.minNodePort = minPort;
         this.maxNodePort = maxPort;
         return self();
     }
 
+    /**
+     * Adds additional commands to the K3s container, e.g. to configure etcd or the K8s API server.
+     *
+     * @param additionalCommands the additional commands
+     * @return <code>this</code>
+     */
+    public SELF withAdditionalCommands(final String... additionalCommands) {
```

Comment:
I'm thinking `withAdditionalCommands` might be a misleading name here since it's just adding cmdline arguments, isn't it? :thinking:

Reply:
We have this function in our own custom K3s container, and I always saw this as a way to customize the `withCommand()` usage from the K3s base container we are extending.

But maybe `withCmdlineOptions()` would be easier to understand?

Reply:
I've renamed the methods to `withCmdlineOptions()` and aligned the JavaDoc and internal naming accordingly.